tool/esm_resolver_generator.js

```diff
@@ -13,7 +13,7 @@ function buildResolver() {
 
 function getModuleNames() {
     let paths = [];
-    var modeNames = modeList();
+    var modeNames = jsFileList("src/mode", /_highlight_rules|_test|_worker|xml_util|_outdent|behaviour|completions/);
```

Comment:
could you change modelist to accept path parameter instead. This is a trickky regexp, better to have it in one place

Reply:
sure, done